src/programming/BL_Python/programming/config/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class InvalidConfigNameError(Exception):
+    """The class name used as a configuration type is invalid. The name must end with `Config`."""
```

Comment:
**suggestion: (non-blocking)** If it this is only going to be used because it has to end with Config and not with any other invalid names, it may be better to reflect this in the name. Maybe something like `NotEndsWithConfigError`. 

If it's going to be a general exception for any invalid config names, then it would be fine to just change the comment and remove the "The name must end with `Config`" in the docstring.